### ISSUE TITLE
/lib/cms_scanner/parsed_cli.rb:34:in `method_missing'

### ISSUE THREAD

**Issue by blshkv** [wpscanteam/CMSScanner#88]
net-analyzer/wpscan-3.5.1
dev-ruby/cms_scanner-0.0.44.0
dev-ruby/opt_parse_validator-0.0.17.1

```
/usr/lib64/ruby/gems/2.4.0/gems/cms_scanner-0.0.44.0/lib/cms_scanner/parsed_cli.rb:34:in `method_missing': undefined method `[]' for nil:NilClass (NoMethodError)
        from /usr/lib64/ruby/gems/2.4.0/gems/cms_scanner-0.0.44.0/lib/cms_scanner/controller.rb:52:in `formatter'
        from /usr/lib64/ruby/gems/2.4.0/gems/cms_scanner-0.0.44.0/lib/cms_scanner/scan.rb:44:in `formatter'
        from /usr/lib64/ruby/gems/2.4.0/gems/cms_scanner-0.0.44.0/lib/cms_scanner/scan.rb:38:in `ensure in run'
        from /usr/lib64/ruby/gems/2.4.0/gems/cms_scanner-0.0.44.0/lib/cms_scanner/scan.rb:38:in `run'
        from /usr/lib64/ruby/gems/2.4.0/gems/wpscan-3.5.1/bin/wpscan:16:in `block in <top (required)>'
        from /usr/lib64/ruby/gems/2.4.0/gems/cms_scanner-0.0.44.0/lib/cms_scanner/scan.rb:15:in `initialize'
        from /usr/lib64/ruby/gems/2.4.0/gems/wpscan-3.5.1/bin/wpscan:6:in `new'
        from /usr/lib64/ruby/gems/2.4.0/gems/wpscan-3.5.1/bin/wpscan:6:in `<top (required)>'
        from /usr/bin/wpscan:9:in `load'
        from /usr/bin/wpscan:9:in `<main>'
```

any idea what's wrong?

**Comment by erwanlr** [comment]
Which command did you use ?

**Comment by erwanlr** [comment]
Fixed in cms_scanner 0.0.44.1
